samples/mixed/consumer/src/main.rs

```diff
@@ -71,11 +95,20 @@ fn start_show_notification_repeater(provider_uri: String, consumer_uri: String)
 /// `provider_uri` - The provider_uri.
 fn start_activate_air_conditioning_repeater(provider_uri: String) {
     debug!("Starting the Consumer's activate-air-conditioning repeater.");
+
     tokio::spawn(async move {
         let mut is_active = true;
+
         loop {
             info!("Sending a set request for entity id {} to the value '{is_active}' to provider URI {provider_uri}",
-                sdv::vehicle::cabin::hvac::is_air_conditioning_active::ID);
+                sdv::hvac::is_air_conditioning_active::ID);
+
+            let metadata: Metadata =
```

Comment:
the type annotations here and in the following statement are not necessary

Reply:
I prefer to make it explicit. Leaving it as is.